app/Notifications/Tiers/TierCreatedNotification.php

```
@@ -0,0 +1,36 @@
+    use Queueable;
+
+    public function __construct(public Tiers $tier)
+    {
+    }
+
+    public function via($notifiable): array
+    {
+        return ['database'];
+    }
+
+    public function toDatabase($notifiable): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$notifiable'. (undefined)

(UnusedFormalParameter)